flask_restful_auth/default_config.py

```diff
@@ -12,7 +12,8 @@
 JWT_ENABLE = True
 """"""
 
-JWT_COOKIE_NAME = 'token'
+JWT_COOKIE_NAME = 'Atoken'
+JWT_NEW_COOKIE_NAME = 'Rtoken'
```

Comment:
I don't understand what `Atoken` or `Rtoken` means. Can we use full names for variables? Also with cookie names, let's use snake case.

Reply:
Access tokens 
and refresh tokens